kge/model/kge_model.py

```diff
@@ -409,7 +415,14 @@ def load_from_checkpoint(
             if not config.log_folder:
                 config.log_folder = "."
         if dataset is None:
-            dataset = Dataset.load(config, preload_data=False)
+            if checkpoint["type"] == "package":
+                dataset = Dataset(config)
+                dataset._meta["entity_ids"] = checkpoint["entity_ids"]
```

Comment:
I suggest to make this more generic. dataset.save_meta(cp, keys) and dataset.load_meta(cp). Store under "dataset.meta.xxx".

Reply:
fixed